envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/RoutesAssertions.kt

```diff
@@ -60,8 +61,12 @@ fun RouteConfiguration.hasNoResponseHeaderToAdd(key: String): RouteConfiguration
 
 fun VirtualHost.hasStatusVirtualClusters(): VirtualHost {
     return this.hasVirtualClustersInOrder(
-        { it.pattern == "/status/.*" && it.name == "status" },
-        { it.pattern == "/.*" && it.name == "endpoints" }
+        { it.headersList.contains(HeaderMatcher.newBuilder()
+                .setName(":path").setPrefixMatch("/status/").build()) && it.name == "status"
+        },
+        { it.headersList.contains(HeaderMatcher.newBuilder()
```

Comment:
shouldn't it be `==`, not `contains`? I think we should not have any other headers here